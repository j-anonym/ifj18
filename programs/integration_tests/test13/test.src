def y(a,b,c)
b=5
c=7
x=c+b
if (a == nil) then
print "param a je nil\n"
else
end
print x, "\n"
end


y(nil,5,6)
