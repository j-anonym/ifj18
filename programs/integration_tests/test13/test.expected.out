param a je nil
12
